src/Solver/Dynamic/Ida.hpp

```diff
@@ -150,9 +150,9 @@ namespace AnalysisManager
       void*           solver_{};
       SUNContext      context_{};
       SUNMatrix       JacobianMat_{};
-      SUNMatrix       JacobianMatB_{};
+      SUNMatrix       JacobianMatB_ = nullptr;
       SUNLinearSolver linearSolver_{};
-      SUNLinearSolver linearSolverB_{};
+      SUNLinearSolver linearSolverB_ = nullptr;
```

Comment:
Either make all of these `nullptr` or make no changes.

Reply:
done. removed the `= nullptr` bit and tested it to ensure it worked.